en/Basic Math/Finding_number_of_Digits_in_a_Number.md

```diff
@@ -0,0 +1,112 @@
+# Finding number of Digits in a Number
+
+# Finding the number of digits in a number
+
+Given an integral number N., the task is to find the count of digits present in this number.
+
+Let's say:
```

Comment:
```suggestion
Let's say:

```

Reply:
Didn't understand, what I've to change here?